components/treeview/stories/treeview.stories.js

```diff
@@ -44,8 +46,149 @@ export default {
 	},
 };
 
+/**
+ * By default, treeviews span the entire width of their parent container and are used within panels.
+ */
 export const Default = TreeViewGroup.bind({});
 Default.args = {
+	items: [
+		{
+			id: "label1",
+			label: "Label 1",
+			link: "#",
+		},
+		{
+			id: "group1",
+			label: "Group 1",
+			link: "#",
+			isOpen: true,
+			items: [
+				{
+					id: "label2",
+					label: "Label 2",
+					link: "#",
+				},
+				{
+					id: "label3",
+					label: "Label 3",
+					link: "#",
+				},
+			],
+		},
+		{
+			id: "group2",
+			label: "Group 2",
+			link: "#",
+			items: [
+				{
+					id: "label3",
+					label: "Label 3",
+					link: "#",
+				},
+				{
+					id: "group3",
+					label: "Group 3",
+					link: "#",
+					items: [
+						{
+							id: "label4",
+							label: "Label 4",
+							link: "#",
+						},
+						{
+							id: "group4",
+							label: "Group 4 (Empty)",
+							link: "#",
+							items: [],
+						},
+					],
+				},
+			],
+		},
+	],
+};
+
+// ********* DOCS ONLY ********* //
+export const Sizing = SizingGroup.bind({});
+Sizing.args = Default.args;
+Sizing.tags = ["autodocs", "!dev"];
+Sizing.parameters = {
+	chromatic: { disableAllSnapshots: true },
```

Comment:
I believe all of these should be `disableSnapshot: true`, rather than `disableAllSnapshots: true`.